src/Tribe/Venue.php

```diff
@@ -297,6 +297,19 @@ public function save( $id, $data, $post_type, $post_status ) {
 	 *
 	 */
 	public function save_meta( $venue_id, $data ) {
+		$venue = get_post( $venue_id );
+
+		/**
+		 * Allow hooking in prior to updating meta fields.
+		 *
+		 * @param int     $venue_id The venue ID we are modifying meta for.
+		 * @param array   $data     The meta fields we want saved.
+		 * @param WP_Post $venue    The venue itself.
+		 *
+		 * @since TBD
+		 */
+		do_action( 'tribe_events_venue_save', $venue_id, $data, $venue );
```

Comment:
Same Q here - could a means of filtering data be more useful or a nice addition to this action hook?

Reply:
Ditto